package.json

```diff
@@ -6,7 +6,8 @@
     "start": "nx serve",
     "build": "nx build",
     "test": "nx test",
-    "postinstall": "husky install && yarn tsc -b tools/executors/next && yarn tsc -b tools/executors/webpack"
+    "postinstall": "husky install && yarn tsc -b tools/executors/next && yarn tsc -b tools/executors/webpack",
+    "test-all": "nx run-many --all --target=test"
```

Comment:
nitpick: can we make this `test:all` I prefer using colons for commands that have subtle differences